src/errors/runtime/message.js

```diff
@@ -24,6 +24,8 @@ export default {
     testNameIsNotAString:                           'The test name is expected to be a string, but it was {type}.',
     testBodyIsNotAFunction:                         'The test body is expected to be a function, but it was {type}.',
     beforeEachIsNotAFunction:                       `Fixture's "beforeEach" method takes a function, but {type} was passed.`,
+    invalidAuthCredentialsFormat:                   `Fixture's "httpAuth" method accepts a credentials object with the "username" and "password" fields.`,
+    authCredentialsIsNotObject:                     `{type} was sent to the fixture's "httpAuth" method instead of an object.`,
```

Comment:
Maybe it's better to use our template as in other similar messages?
`Fixture's "httpAuth" method takes an object, but {type} was passed.`